test/local/routes/emails.js

```diff
@@ -723,12 +723,9 @@ describe('/recovery_email', () => {
     it('should fail with unverified primary email', () => {
       route = getRoute(accountRoutes, '/recovery_email')
       mockRequest.auth.credentials.emailVerified = false
-      return runTest(route, mockRequest, () => {
-        assert.fail(new Error('Should have failed adding secondary email with unverified primary email'))
+      return runTest(route, mockRequest).then(() => assert.fail('Should have failed adding secondary email with unverified primary email'), err => {
+        assert.equal(err.errno, 104, 'unverified account')
```

Comment:
I'm bike-shedding, but I find the formatting here a little bit hard to read. Not sure if it would be cleaner if both promise handlers were on their own line?